scripts/buildSampleTree.ts

```diff
@@ -14,15 +14,15 @@ const POOL_REBALANCE_LEAF_COUNT = 1;
 const RELAYER_REFUND_LEAF_COUNT = 1;
 const SLOW_RELAY_LEAF_COUNT = 1;
 const POOL_REBALANCE_NET_SEND_AMOUNT = 0.1; // Amount of tokens to send from HubPool to SpokePool
-const RELAYER_REFUND_AMOUNT_TO_RETURN = 0.1; // Amount of tokens to send from SpokePool to HubPool
-const L1_TOKEN = "0x40153ddfad90c49dbe3f5c9f96f2a5b25ec67461";
-const L2_TOKEN = "0x40153ddfad90c49dbe3f5c9f96f2a5b25ec67461";
+const RELAYER_REFUND_AMOUNT_TO_RETURN = 0; // Amount of tokens to send from SpokePool to HubPool
```

Comment:
nit: no need to modify this file, just keep changes local

Reply:
Holdover from testing: Done https://github.com/across-protocol/contracts-v2/pull/381/commits/9a80eb7b249186281c3abb5514d8b30eadf5f74e